packages/openapi-typescript/bin/cli.js

```diff
@@ -11,26 +11,27 @@ const HELP = `Usage
   $ openapi-typescript [input] [options]
 
 Options
-  --help                     Display this
-  --version                  Display the version
-  --redocly [path], -c       Specify path to Redocly config (default: redocly.yaml)
-  --output, -o               Specify output file (if not specified in redocly.yaml)
-  --enum                     Export true TS enums instead of unions
-  --enum-values              Export enum values as arrays
-  --dedupe-enums             Dedupe enum types when \`--enum=true\` is set
-  --check                    Check that the generated types are up-to-date. (default: false)
-  --export-type, -t          Export top-level \`type\` instead of \`interface\`
-  --immutable                Generate readonly types
-  --additional-properties    Treat schema objects as if \`additionalProperties: true\` is set
-  --empty-objects-unknown    Generate \`unknown\` instead of \`Record<string, never>\` for empty objects
-  --default-non-nullable     Set to \`false\` to ignore default values when generating non-nullable types
+  --help                                      Display this
+  --version                                   Display the version
+  --redocly [path], -c                        Specify path to Redocly config (default: redocly.yaml)
+  --output, -o                                Specify output file (if not specified in redocly.yaml)
+  --enum                                      Export true TS enums instead of unions
+  --enum-values                               Export enum values as arrays
+  --dedupe-enums                              Dedupe enum types when \`--enum=true\` is set
+  --check                                     Check that the generated types are up-to-date. (default: false)
+  --export-type, -t                           Export top-level \`type\` instead of \`interface\`
+  --immutable                                 Generate readonly types
+  --additional-properties                     Treat schema objects as if \`additionalProperties: true\` is set
+  --empty-objects-unknown                     Generate \`unknown\` instead of \`Record<string, never>\` for empty objects
+  --default-non-nullable                      Set to \`false\` to ignore default values when generating non-nullable types
   --properties-required-by-default
-                             Treat schema objects as if \`required\` is set to all properties by default
```

Comment:
Minor feedback: I’d like to keep the current column split, and simply wrap to the next line like this. Reason is many people have ~80char terminal windows, and if the columns are too spaced out it’s hard to read. The current column sizing is arbitrary and can be adjusted a little bit, but IMO the adjustment here spaces them out too much

Reply:
Done.